src/manifest/assertions/IngredientAssertion.ts

```diff
@@ -103,6 +132,101 @@ export class IngredientAssertion extends Assertion {
         return this.documentID === undefined && this.validationStatus === undefined && this.c2pa_manifest === undefined;
     }
 
+    private serializeV1(claim: Claim): RawIngredientMapV1 {
+        if (!this.relationship) throw new Error('Assertion has no relationship');
+
+        const content: RawIngredientMapV1 = {
+            'dc:title': this.title!,
```

Comment:
These methods seem to be missing a check that these mandatory fields have been set